data/json/items/comestibles/seed.json

```diff
@@ -14,6 +14,23 @@
         "symbol": ".",
         "color": "brown"
     },
+    {
+        "id": "seed_bee_balm",
+        "copy-from": "seed",
+        "type": "COMESTIBLE",
+        "name": "bee balm seeds",
+        "name_plural": "bee balm seeds",
+        "description": "Some bee balm seeds.  You could probably plant these.",
+        "charges": 2,
+        "material": "veggy",
```

Comment:
@mugling I was thinking about changing it to veggy actually. There are only like 8 fruit and a few wheat material seeds. For the color I think green is the better color of the two there. For charges do they really need charges? Better to be single use and have them stack.